DataFormats/Parameters/src/GRPObject.cxx

```diff
@@ -66,3 +68,21 @@ void GRPObject::print() const
     printf("\n");
   }
 }
+
+//_______________________________________________
+GRPObject* GRPObject::loadFrom(const std::string grpFileName, std::string grpName)
+{
+  // load object from file
+  TFile flGRP(grpFileName.data());
+  if (flGRP.IsZombie()) {
+    LOG(ERROR) << "Failed to open " << grpFileName << FairLogger::endl;
+    return nullptr;
+  }
+  auto grp =
+    static_cast<o2::parameters::GRPObject*>(flGRP.GetObjectChecked(grpName.data(), o2::parameters::GRPObject::Class()));
```

Comment:
shouldn't that be `reinterpret_cast`? We want the compiler simply to treat the pointer as the specified one, nothing else.